src/protocol/resp/zcount.rs

```diff
@@ -0,0 +1,73 @@
+// Copyright 2025 Pelikan Foundation LLC.
+// Licensed under the Apache License, Version 2.0
+// http://www.apache.org/licenses/LICENSE-2.0
+
+use std::io::Write;
+use std::time::Duration;
+
+use momento::cache::{SortedSetLengthByScoreRequest, SortedSetLengthByScoreResponse};
+use momento::CacheClient;
+use protocol_resp::{SortedSetCount, ZCOUNT, ZCOUNT_EX, ZCOUNT_HIT, ZCOUNT_MISS};
+use tokio::time;
+
+use crate::error::ProxyResult;
+use crate::klog::{klog_1, Status};
+use crate::ProxyError;
+
+use super::update_method_metrics;
+
+pub async fn zcount(
+    client: &mut CacheClient,
+    cache_name: &str,
+    response_buf: &mut Vec<u8>,
+    req: &SortedSetCount,
+) -> ProxyResult {
+    update_method_metrics(&ZCOUNT, &ZCOUNT_EX, async move {
+        let min_score = match (req.min_score(), req.min_score_exclusive()) {
+            (f64::NEG_INFINITY, _) => None,
+            (f64::INFINITY, _) => Some(f64::MAX), // Momento does not accept +inf, but accepts max f64
+            (score, true) => Some(score + 1.0),
```

Comment:
The `SortedSetLengthByScore` rpc on Momento supports specifying inclusive vs exclusive bounds. So we should update the Rust SDK to support those and not add small values as a workaround.